src/forces/Force.re

```diff
@@ -0,0 +1,23 @@
+type entity = {
+  acceleration: Vector.t(float),
+  velocity: Vector.t(float),
+  position: Vector.t(float),
+};
+
+let applyForce = (~force, ~moverMass, ~acceleration, ~velocity, ~position) => {
+  // Derive the acceleration created by the force and add it to the current acceleration.
+  let nextAcceleration =
+    Vector.addf(~v1=acceleration, ~v2=Vector.divf(~v=force, ~s=moverMass));
+
+  // Add the acceleration to the current velocity.
+  let nextVelocity = Vector.addf(~v1=velocity, ~v2=nextAcceleration);
+
+  // Add the velocity to the position.
+  let nextPosition = Vector.addf(~v1=position, ~v2=nextVelocity);
+
+  {
+    acceleration: nextAcceleration,
+    velocity: nextVelocity,
+    position: nextPosition,
+  };
+};
```

Comment:
The money maker 🤑 This is the function that determines, given an object with velocity, acceleration, and position, what will its _next_ velocity, acceleration, and position be? Thinking now that I should pass in the `entity` as a record directly to this function.